src/zope/app/apidoc/codemodule/browser/README.rst

```diff
@@ -44,7 +44,7 @@ Return info objects for all classes in this module.
   >>> pprint(details.getClasses())
   [{'doc': 'Represent the code browser documentation root',
     'name': 'CodeModule',
-    'url': 'http://127.0.0.1/++apidoc++/Code/zope/app/apidoc/codemodule/codemodule/CodeModule'}]
+    'url': 'http://127.0.0.1/++etc++site/++apidoc++/Code/zope/app/apidoc/codemodule/codemodule/CodeModule'}]
```

Comment:
It *was* mostly due to a different rootFolder between the two setups. I fixed that.

But I also had to make an adjustment to fix the issue with zope.securitypolicy, by always attaching *directly* to a (phony) root when we traverse through ++apidoc++. This is correct to make the tests all pass, but it may not be correct in general? I'm not sure how it would work with virtual hosts or multiple distinct sites in a single instance...